.github/workflows/Test-AITs.yml

```diff
@@ -32,6 +37,12 @@ on:
         required: false
         default: 'main'
         type: string
+      single-test:
```

Comment:
I don't think this input is needed in workflow_call.
Right now the use of workflow_call is to run all AITs in the test suite.

Do you see any use case for another workflow run a single AIT?
Or is this necessary because the input is referenced in the code?

Reply:
I will remove it then